### DISCUSSION TITLE
Column name return "validation.{columnname}"

### DISCUSSION THREAD

**Discussion by vincentmikhael** [Okipa/laravel-table#114] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/61272406/202623820-fd7e2f8f-37ab-49df-a2ad-bdfb862e45cf.png)

i just want show column name only without "validation. " text

**Comment by Okipa** [comment, marked as answer]
@vincentmikhael You just have to use the `->title()` available column method, as specified on the doc here: https://github.com/Okipa/laravel-table#declare-columns-on-tables
